tardis/tardis_portal/api.py

```diff
@@ -872,3 +884,20 @@ def hydrate(self, bundle):
             bundle.data['file_object'].close()
             del(bundle.data['file_object'])
         return bundle
+
+
+class ObjectAclResource(MyTardisModelResource):
+    content_object = GenericForeignKeyField({
+        Experiment: ExperimentResource,
+        # ...
+    }, 'content_object')
+
+    class Meta:
+        authentication = default_authentication
+        authorization = ACLAuthorization()
+        queryset = ObjectACL.objects.all()
+        filtering = {
+            'object_id': ('exact', ),
```

Comment:
The object_id is a but meaningless without the content_type.
Either we don't allow querying on the id or we support setting the content_type. This would require setting the foreign key field to a ContentTypeResource.